src/components/Hero/Hero.jsx

```diff
@@ -1,27 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Button from '../Button/Button';
+import { share, heart } from '../SVG/SVG';
 import styles from './Hero.css';
 
-const Hero = ({ testImgSrc }) =>
+const Hero = ({ heroImgURI, modalGalleryHandler, modalShareSaveHandler }) =>
   (
-    <div className={styles.container} style={{ backgroundImage: `url(${testImgSrc})` }}>
+    <div
+      className={styles.container}
+      style={{ backgroundImage: `url(${heroImgURI})` }}
+      onClick={modalGalleryHandler}
+      role="presentation"
+    >
       <div className={styles['upper-button-box']}>
-        <Button />
-        <Button />
+        <Button icon={share} label="Share" clicked={modalShareSaveHandler} />
```

Comment:
Is `icon` a `<button>` attribute? 
Also, what is `clicked`? Is that also a `<button>` native attribute?

Reply:
Neither are native button attributes. They are props.

icon receives an SVG.
clicked receives a method.

Do you think I should change the names or keep them as they are?  If you think I should change the names, do you have any suggestions?

Reply:
No actually, now that we went through it, it makes sense to me. All good!